subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/finalizer/InstrumentationTestActionFinalizerImpl.kt

```diff
@@ -13,12 +13,18 @@ internal class InstrumentationTestActionFinalizerImpl(
     private val buildFailer: BuildFailer,
     private val verdictFile: File,
     private val verdictDeterminer: VerdictDeterminer,
+    private val finalizerFileDump: FinalizerFileDump,
 ) : InstrumentationTestActionFinalizer {
 
     override fun finalize(testSchedulerResults: TestsScheduler.Result) {
 
         val initialTestSuite: Set<TestStaticData> = testSchedulerResults.testSuite.testsToRun.map { it.test }.toSet()
 
+        finalizerFileDump.dump(
+            initialTestSuite = initialTestSuite,
+            testResults = testSchedulerResults.testResults
+        )
```

Comment:
How did you choose what to include in this dump? I don't quite understand how it will be used.
E.g. why without `Verdict`?

Reply:
i added debug information for particular case: see task
i'll add a Verdict too, but it is already available